docs/concept-exercises.md

```diff
@@ -51,30 +51,51 @@ An exercise has the following files. In the browser they will show at the releva
 
 ### `.docs/introduction.md`
 
-This file contains an introduction to the concept. It should make the exercise's learning goals explicit and provide a short introduction with enough detail for the student to complete the exercise. The aim is to give the student just enough context to figure out the solution themselves, as research has shown that self-discovery is the most effective learning experience. Using the proper technical terms in the descriptions will be helpful if the student wants to search for more information. If the exercise introduces new syntax, an example of the syntax should always be included; students should not need to search the web for examples of syntax.
+**Purpose:** Introduce the concept(s) that the exercise teaches to the student.
 
-As an example, the introduction to a "strings" exercise might describe a string as just a "Sequence of Unicode characters" or a "series of bytes". Unless the student needs to understand more nuanced details in order to solve the exercise, this type of brief explanation (along with an example of its syntax) should be sufficient information for the student to solve the exercise.
+- The information provided should give the student just enough context to figure out the solution themselves. 
```

Comment:
If it provides them with a link that contains required information, is that also valid? Or should we assume their internet isn't working and that they should be able to solve the exercise just with the information in this file?

Reply:
My issue with links in the introduction is that the page linked to will almost always contain loads of *extra* information that overloads the student. If we could avoid that, then I'd be ok with links, and for complex topics (e.g. recursion) there might be great resources to link to. But generally I think explaining things concisely in the introduction is the right thing to do.

Reply:
I agree with that. Could you perhaps add the info from your comment explicitely to the doc?